tests/Unit/IO/H5/CMakeLists.txt

```diff
@@ -39,6 +39,23 @@ target_link_libraries(
   Utilities
   )
 
+spectre_add_python_bindings_test(
+  "Unit.IO.H5.CombineH5.Python"
+  "Test_CombineH5.py"
+  "unit;IO;H5;Python"
+  PyH5
+  )
+
+if(${BUILD_PYTHON_BINDINGS})
+  # Test is a bit slow because it writes a bunch of plot files to verify
+  # the argument handling works.
+  set_tests_properties(
+    "Unit.IO.H5.CombineH5.Python"
+    PROPERTIES
+    TIMEOUT 80
```

Comment:
This is too much :D Maybe 10 seconds would be ok if you write to files. The test doesn't look so long, maybe just try 10 seconds as see if it's enough for CI?